SportsResults.frockett/SportsResults.frockett/EmailService.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Net;
+using System.Net.Mail;
+
+namespace SportsResults.frockett;
+
+internal class EmailService
+{
+    // SMPT Address and port number are examples
+    static string smptAddress = "smtp.gmail.com";
```

Comment:
Since the email config are hard coded here it's impossible for the user to change who they want your service to send emails to or even from which email it is being sent from or email provider :sweat_smile: 

Consider reading them from a config file or appsettings.json

Reply:
I hardcoded them since it seemed that the main purpose of the project was the scraping. I'll leave them hardcoded for now. I based my design off the requirement that the program need not be interactable. If I misunderstood the requirements, let me know.

Reply:
Well I suggested that change because I believe it was something you learned from previous projects and so I expected to see that implemented. But yeah you are right, it wasn't explicitly stated in the requirements so it's not a big deal :wink: